src/test/suite/mdbExtensionController.test.ts

```diff
@@ -1670,7 +1655,7 @@ suite('MDBExtensionController Test Suite', function () {
 
     let calledDelete = false;
 
-    const mockDataService: DataService = {
+    const mockDataService: any = {
```

Comment:
Does something prevent us from keep using `DataService` type here?

Reply:
Answered in https://github.com/mongodb-js/vscode/pull/471#discussion_r1100342425